packages/components/psammead-section-label/src/__snapshots__/index.test.jsx.snap

```diff
@@ -53,6 +52,7 @@ exports[`SectionLabel When hideSectionHeader is true should add styling to hide
 .c0 {
   position: relative;
   z-index: 0;
+  color: #3F3F42;
```

Comment:
I can't seem to see the other colour listed in this snapshots. Do we have a test case for it?

Reply:
I can see `C_EBON` being used [here](https://github.com/bbc/psammead/blob/b66faf72639cc1b5fe11d46fe48e16589e91936c/packages/components/psammead-section-label/src/__snapshots__/index.test.jsx.snap#L155).

Reply:
Thanks, Fahad!